### PR TITLE
Db 382 python 3

### PR DESCRIPTION
**Pull request by nmonga91**
Updating code to be compatible with python 3

### CODE REVIEW

**Comment by nmonga91** [comment]
The build is failing because `import urllib.parse` is only in Python 3 and the build is running Python 2.

File: dataactbroker/handlers/jobHandler.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def monthToQuarter(cls, month, isStart):
             raise ResponseException("Not a valid month to be in quarter format", StatusCode.INTERNAL_ERROR, ValueError)
         quartersFromStart = monthsIntoFiscalYear / 3
         quarter = quartersFromStart + 1
-        return "".join(["Q",str(quarter)])
+        return "".join(["Q",str(int(quarter))])
```
Comment (jworcestBAH) [inline dataactbroker/handlers/jobHandler.py]: Good catch, We don't want to report things as Q1.333

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by jworcestBAH** [comment]
Merging

**Pull request merged**